apps/pdc-frontend/src/styles/openforms.css

```diff
@@ -0,0 +1,11 @@
+.utrecht-form-container--openforms .utrecht-textbox--openforms:disabled {
+  /* color tokens werken niet */
+  --utrecht-textbox-disabled-background-color: hsl(0 0% 90%);
+  --utrecht-textbox-disabled-border-color: transparent;
+  --utrecht-textbox-disabled-color: hsl(0 0% 20%);
+}
+
+.formio-editor-read-only-content {
+  white-space: pre-line;
+  white-space-collapse: preserve-breaks;
```

Comment:
Zou je willen proberen de mixin te gebruiken uit `@utrecht/components/.....`?

Reply:
ik denk niet dat het gaat lukken om de mixin te gebruiken want de components map zelf staan niet in de node modules. Dus je kan dan zelf geen mixin inporten en daar van de styling gebruiken.

Reply:
@scar055 Je kan daarvoor `@utrecht/components` installeren, die bevat de mixins voor elk component

Reply:
alles klopt nu volgens mij en het werkt